packit_service/worker/tasks.py

```diff
@@ -132,15 +132,11 @@ class BodhiHandlerTaskWithRetry(HandlerTaskWithRetry):
 @celery_app.task(
     name=getenv("CELERY_MAIN_TASK_NAME") or CELERY_DEFAULT_MAIN_TASK_NAME, bind=True
 )
-def process_message(
-    self, event: dict, topic: str = None, source: str = None
-) -> List[TaskResults]:
+def process_message(self, event: dict) -> List[TaskResults]:
     """
```

Comment:
Could you please also switch it to the google style docstrings while you're making changes here?